docs/en/notes/changelog.md

```diff
@@ -1,5 +1,53 @@
 # Changelog of v3.x
 
+## v3.0.0rc2 (21/10/2022)
+
+### Highlights
+
+- Support [imagenet pre-training](configs/rtmdet/cspnext_imagenet_pretrain) for RTMDet's backbone
+
+#### New Features
+
+- Support [imagenet pre-training](configs/rtmdet/cspnext_imagenet_pretrain) for RTMDet's backbone (#8887)
+- Add `CrowdHumanDataset` and Metric (#8430)
+- Add `FixShapeResize` to support resize of fixed shape (#8665)
+
+#### Bug Fixes
+
+- Fix `ConcatDataset` Import Error (#8909)
+- Fix `CircleCI` and `readthedoc` build failed (#8980, #8963)
+- Fix bitmap mask translate when `out_shape` is different (#8993)
+- Fix inconsistency in `Conv2d` weight channels (#8948)
+- Fix bugs when plotting loss curve by analyze_logs.py (#8944)
+- Fix type change of labels in `albumentations` (#9074)
+- Fix some docs and types error (#8818)
+
+#### Improvements
+
+- Refactored standard roi head with `box type` (#8658)
+- Support mask concatenate in `BitmapMasks` and `PolygonMasks` (#9006)
```

Comment:
concatenation

Reply:
We should also tell users that we bump the dependency of the version of mmengine

Reply:
There is no change in the `mmengine` version. Just changed the upper limit.